build_scripts/appimage/install-infection-monkey-service.sh

```diff
@@ -65,9 +65,9 @@ uninstall_service() {
 move_appimage() {
   sudo mkdir --mode=0755 -p "${MONKEY_BIN}"
 
-  if [ "$1" != "${MONKEY_BIN}/${APPIMAGE_NAME}" ] ; then
+  if [ "${APPIMAGE}" != "${MONKEY_BIN}/${APPIMAGE_NAME}" ] ; then
```

Comment:
I wasn't 100% sure about this check in the last round, and I'm still not. Is there a downside to just overwriting what's already there? I could see this potentially leading to issues during development. At a minimum, we should print some kind of message so that the user knows that the destination file already existed and wasn't overwitten.

Reply:
`touch a && cp a a` has exit code 1 (and can't be forced afaik) thus failing the script due to `set -e`

can be altered by having `set -e` removed, using `set +e` before `cp`, or keeping the if

Reply:
Ok. This names sense. I misread the condition.